docs/user-guide/options.md

```diff
@@ -84,7 +84,7 @@ Specify the formatter to format your results.
 Options are:
 
 - `compact` - generates output similar to ESLint's compact formatter
-- `github` - generates messages via [workflow commands for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions)
```

Comment:
@ybiquitous I added a `(DEPRECATED)` at the end of the string.
I also changed the same string in `cli.mjs` so it is more visible.